goodmap/core_api.py

```diff
@@ -15,12 +15,13 @@ def core_pages(database, languages, email_service):
     core_api_blueprint = Blueprint('api', __name__, url_prefix="/api")
     core_api = Api(core_api_blueprint, doc='/doc', version='0.1')
 
-    @core_api.route("/locations:report", methods=["POST"])
-    @expects_json(report_location_schema)
-    def report_location():
-        request_body = request.get_json()
-        email_service.send_report_location_email(request_body)
-        return "", 200
+    if email_service:
```

Comment:
I see a big misunderstanding here. What I meant was only to remove it from test config.
As this is an integral part of code.